cmd/metal-api/main.go

```diff
@@ -600,11 +600,11 @@ func enrichSwaggerObject(swo *spec.Swagger) {
 			},
 			License: &spec.License{
 				LicenseProps: spec.LicenseProps{
-					Name: "MIT",
-					URL:  "http://mit.org",
+					Name: "AGPL-3.0",
+					URL:  "https://www.gnu.org/licenses/agpl-3.0.de.html",
 				},
 			},
-			Version: "1.0.0",
+			Version: v.Version,
```

Comment:
This does not work unfortunately because we call `make spec` manually and push the result then. The version tag is incorrect at this point in time though... don't know how we can handle this more adequately. :(

Reply:
We decided on just not putting the version in there.